user_docs/en/userGuide.md

```diff
@@ -2163,6 +2163,10 @@ This means that you do not have to scroll the display at the end of each line ev
 This may allow for more fluent reading of large amounts of text.
 It is disabled by default.
 
+##### Show paragraphs start {#BrailleSettingsShowParagraphStart}
+
```

Comment:
Could you add a table with the possible options and the default one, as done for any recently introduced feature flag option?

Reply:
Done.